src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -2701,36 +2708,37 @@ void setCredentialsId (final String id) {
          */
         @Deprecated
         public ModuleLocation(String remote, String local, String depthOption, boolean ignoreExternalsOption) {
-            this(remote,null,local,depthOption,ignoreExternalsOption);
+            this(remote,null,local,depthOption,true,ignoreExternalsOption);
         }
 
         @DataBoundConstructor
-        public ModuleLocation(String remote, String credentialsId, String local, String depthOption, boolean ignoreExternalsOption) {
+        public ModuleLocation(String remote, String credentialsId, String local, String depthOption, boolean stickyDepth, boolean ignoreExternalsOption) {
```

Comment:
Thank you! I'm quite new to Java :-)
I've pushed a fix commit to my repository. Now, how is it customary to
proceed in such cases? Shall I open a new pull request?

On Wed, Sep 9, 2015 at 12:32 PM, Oleg Nenashev <notifications@github.com>
wrote:

> In src/main/java/hudson/scm/SubversionSCM.java
> <https://github.com/jenkinsci/subversion-plugin/pull/134#discussion_r39023278>
> :
>
> >          }
> >
> >          @DataBoundConstructor
> > -        public ModuleLocation(String remote, String credentialsId, String local, String depthOption, boolean ignoreExternalsOption) {
> > +        public ModuleLocation(String remote, String credentialsId, String local, String depthOption, boolean stickyDepth, boolean ignoreExternalsOption) {
>
> binary compatibility issue. It's better to keep and deprecate the original
> constructor
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/subversion-plugin/pull/134/files#r39023278>.
>